guides/source/development_dependencies_install.md

```diff
@@ -165,7 +165,7 @@ $ bundle exec ruby -Itest path/to/test.rb -n test_name
 
 ### Active Record Setup
 
-The test suite of Active Record attempts to run four times: once for SQLite3, once for each of the two MySQL gems (`mysql` and `mysql2`), and once for PostgreSQL. We are going to see now how to set up the environment for them.
+Active Record's test suite runs three times: once for sqlite3, once for mysql2, and once for postgresql. We are going to see now how to set up an environment for each of them.
```

Comment:
I prefer the previous phrasing here: we use the official software names where possible, and "the environment" is more accurate.